azure/src/services/azureFunctionCloudService.test.ts

```diff
@@ -3,6 +3,7 @@ import {
   ContainerResolver,
   CloudContainer,
   CloudService,
+  InvokeRequest,
```

Comment:
Azure changes must be in another PR, leave this PR just for `core`

Reply:
Thanks for this feedback, now it only have the core.